eng/common/TestResources/New-TestResources.ps1.md

```diff
@@ -35,8 +35,9 @@ New-TestResources.ps1 [-BaseName <String>] [-ResourceGroupName <String>] [-Servi
 Deploys live test resouces specified in test-resources.json files to a resource
 group.
 
-This script searches the directory specified in $ServiceDirectory recursively
-for files named test-resources.json.
+This script searches for files named test-resources.json in each artifact project
+of $Artifacts under $ServiceDirectory recursively. When $Artifacts is not set, the
```

Comment:
Sometimes directory name not equal to artifact name.
So maybe we can use `SubDirectories` instead of `Artifacts`?

Reply:
![image](https://user-images.githubusercontent.com/13167207/105809947-0fabd100-5fe5-11eb-9abf-5707d5d387f5.png)